internal/service/organization/resource_organization.go

```diff
@@ -104,7 +104,7 @@ func resourceMongoDBAtlasOrganizationCreate(ctx context.Context, d *schema.Resou
 		PublicKey:        *organization.ApiKey.PublicKey,
 		PrivateKey:       *organization.ApiKey.PrivateKey,
 		BaseURL:          meta.(*config.MongoDBClient).Config.BaseURL,
-		TerraformVersion: meta.(*config.Config).TerraformVersion,
+		TerraformVersion: meta.(*config.MongoDBClient).Config.TerraformVersion,
```

Comment:
is this happening in other resources?

Reply:
No, this was introduced [here](https://github.com/mongodb/terraform-provider-mongodbatlas/commit/ac9b66be6ed10fbd9022a2ec53d52ab3ba9fc09f) and there are no other cases of this in the code